doc/release-notes-26618.md

```diff
@@ -0,0 +1,4 @@
+RPC Wallet
+----------
+
+- RPC `unloadwallet` now fails if a rescan is in progress. (#26618)
```

Comment:
26618: might want to rename the file and tweak in this line to ref this PR instead

Reply:
```suggestion
- RPC `unloadwallet` now fails if a rescan is in progress. (#6759)
```